sycl/doc/extensions/experimental/sycl_ext_oneapi_virtual_mem.asciidoc

```diff
@@ -296,7 +296,9 @@ for the device associated with this instance of `physical_mem`.
 If `mode` is `address_access_mode::read` or `address_access_mode::read_write`
 the returned pointer is accessible after the call as read-only or read-write
 respectively. Otherwise, it is considered inaccessible and accessing it will
-result in undefined behavior.
+result in undefined behavior. The accessibility applies to all devices in
+the context associated with this instance of `physical_mem` from which the
+physical memory is reachable.
```

Comment:
Under what circumstances would the physical memory be reachable from another device?  I assume this would only be the case if the application used [sycl_ext_oneapi_peer_access](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_oneapi_peer_access.asciidoc) to enable peer-to-peer access?  If that is the case, I think we should word it more explicitly:

> If other devices have enabled peer-to-peer access with this device via [sycl_ext_oneapi_peer_access](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_oneapi_peer_access.asciidoc), those other devices also have access to this mapped memory, and the access mode applies to accesses from those devices also.